docker/Dockerfile.release

```diff
@@ -0,0 +1,69 @@
+# This dockerfile is used for quay.io images
+FROM ubuntu:bionic as builder
+
+ARG bcc_ref="v0.12.0"
+ARG LLVM_VERSION="8"
+ENV LLVM_VERSION=$LLVM_VERSION
+
+# There is no need need for most of this once
+# https://github.com/iovisor/bcc/pull/2677 merges, as this is all to build
```

Comment:
https://github.com/iovisor/bcc/pull/2677 looks merged

Reply:
Sorry comment should say “Released”. The packages on the iovisor apt repo are also out of date. Once bcc 0.13.0 is out we shouldn’t need to build it to get all of the static libs, but for now we still do.

I’ll update the comment.